azure-pipelines/templates/basic-setup-steps.yml

```diff
@@ -21,5 +21,5 @@ steps:
 - script: pip install --upgrade -r requirements.txt
   displayName: 'Install requirements'
 
-- script: pip install -e .
+- script: pip install .
   displayName: 'Install from Source'
```

Comment:
Is this comment still right if we drop `-e`?

Reply:
We no longer are dropping the `-e .`. I identified a deeper issue with the recent pip updates that I was able to resolve :)